City-of-Pori/pori-kada
# PORI-309 Removed the post date from project liftups

To test:

1. drush fra -y && drush cc all
2. Go to https://local.pori.fi/ and make sure the project liftup doesn't have post date anymore (the last liftup with large image, title, body and link button)
3. Go to https://local.visitpori.fi/yyteri and make sure project liftups here don't jave post dates anymore either.
